src/v2/view-builder/views/IdentifierView.js

```diff
@@ -36,6 +36,25 @@ const Footer = BaseFooter.extend({
 });
 
 export default BaseView.extend({
+  initialize (options) {
+    // add callout for messages
+    this.messages = options.messages;
+    if (this.messages && this.messages.value.length) {
+      this.showMessageCallout(options.messages.value[0].message, 'warning');
+    }
+    BaseView.prototype.initialize.call(this, arguments);
+  },
+  postRender () {
+    // If user enterted identifier is not found, API sends back a message with a link to sign up
+    // This is the click handler for that link
+    if (this.messages && this.messages.value.length) {
```

Comment:
- may not need the check because when `this.$el.find()` finds no elements, it's essentially a no operation.? What do you think?
- The comments explains is great!

Reply:
I didn't want to add the listener at all if the messages object was not present.

Reply:
my understanding is `this.$el.find()` will get empty array if the `js-sign-up` element is not present hence no listener is added.